common/tasks/interleaved_weighted_round_robin.go

```diff
@@ -311,17 +332,14 @@ func (s *InterleavedWeightedRoundRobinScheduler[T, K]) channels() iwrrChannels[T
 }
 
 func (s *InterleavedWeightedRoundRobinScheduler[T, K]) setupDispatchTimer() {
-	throttleDuration := iwrrMinDispatchThrottleDuration +
-		backoff.FullJitter(s.options.MaxDispatchThrottleDuration-iwrrMinDispatchThrottleDuration)
-
 	s.dispatchTimerLock.Lock()
 	defer s.dispatchTimerLock.Unlock()
 
 	if s.dispatchTimer != nil {
-		s.dispatchTimer.Stop()
```

Comment:
Why are we removing this?

Reply:
If there's already a timer, there's no need to create a new one.